### PR TITLE
Added rancher2_multi_cluster_app datasource and resource

### PR DESCRIPTION
**Pull request by rawmind0**

### CODE REVIEW

**Comment by cloudnautique** [comment]
Can this one match up with the UX for cluster apps?

**Comment by rawmind0** [comment]
Yep, i'll update it asap.

**Comment by rawmind0** [comment]
Updated to match up with the UX for cluster apps.

File: website/docs/r/multiClusterApp.html.markdown [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required/ForceNew) The multi cluster app name (string)
+* `roles` - (Required) The multi cluster app roles (list)
+* `targets` - (Required) The multi cluster app target projects (list)
+* `template_version_id` - (Required) The multi cluster app template version ID. Should be in the format `<catalog_name>-<template_name>-<template_version>` (string)
```
Comment (cloudnautique) [inline website/docs/r/multiClusterApp.html.markdown]: This should be updated?

Comment (rawmind0) [reply website/docs/r/multiClusterApp.html.markdown]: Sure, i missed one commit..... Updated

**Review by cloudnautique — APPROVED** [review]

**Pull request merged**